HeinOnline.js

```diff
@@ -0,0 +1,158 @@
+{
+	"translatorID": "3dcbb947-f7e3-4bbd-a4e5-717f3701d624",
+	"label": "HeinOnline",
+	"creator": "Frank Bennett",
+	"target": "^https?://heinonline\.org/HOL/(?:LuceneSearch|Page)\\?",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-07-04 10:32:10"
+}
+
+function detectWeb (doc, url) {
+	if (url.indexOf("/LuceneSearch?") > -1) {
+		return "multiple";
+	} else {
+		return "journalArticle";
+	}
+}
+
+function Data(doc) {
+	this.node = ZU.xpath(doc, '//form[@id="Print1"]');
```

Comment:
I tried both getElementById() and the first node reported from the xpath statement, and both returned an empty array from a known-working xpath statement against this form node. I must be missing something, but I'm not sure what.

Leading the statement with "//" instead of ".//" yields a result, but in other contexts I have seen that grab everything in the document, which would not be good. If I'm missing something basic about xpath syntax, I can change this over. Otherwise, I would feel more comfortable leaving as it is.